isis/src/control/apps/cnethist/main.cpp

```diff
@@ -124,18 +124,23 @@ void IsisMain() {
         
       //  Write histogram in tabular format
       fout << endl;
-      fout << "ResidualMagnitude,MeasuresInBin,CumulativeMeasures,Percent,CumulativePercent" << endl;
+      fout << "ResidualMagnitudeMin,ResidualMagnitudeMax,MeasuresInBin,CumulativeMeasures,Percent,CumulativePercent" << endl;
```

Comment:
I am concerned about how this could change the app output. If this is being parsed by column number you just threw all of the column numbers off. We need to decide how to handle this type of stuff as developers. I'm hesitant to put this out in a bug fix release, but I think it would be okay to put out in a feature release.